zmon-controller-app/src/main/java/org/zalando/zmon/config/SentryProperties.java

```diff
@@ -0,0 +1,16 @@
+package org.zalando.zmon.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "sentry")
+public class SentryProperties {
+
+    private String enabled;
+    private String dsn;
+    private String dsnUI;
+    private String sampleRate;
+    private String environment;
+    private String servername;
+    private String tags;
+
+}
```

Comment:
@vibhory2j i was unable to get the configuration in the javascript environment. Consulted with @rajatparida86 and it seems something is missing. I have added the JS part so as soon as we add that missing part should work. Otherwise let me know!